p2p/libp2p/memMessenger.go

```diff
@@ -11,6 +12,8 @@ import (
 	"github.com/libp2p/go-libp2p/p2p/net/mock"
 )
 
+const targetPeerCount = 100
+
 // NewMemoryMessenger creates a new sandbox testable instance of libP2P messenger
 // It should not open ports on current machine
 // Should be used only in testing!
```

Comment:
arrange the input of NewMemoryMessanger - put the return params in new line.

Reply:
done